catalog/core/catalog-core-validator/src/main/java/ddf/catalog/validation/impl/validator/MatchAnyValidator.java

```diff
@@ -34,11 +34,16 @@
   private final List<AttributeValidator> validators;
 
   public MatchAnyValidator(List<AttributeValidator> validators) {
-    this.validators =
-        validators
-            .stream()
-            .sorted(Comparator.comparingInt(Object::hashCode))
-            .collect(Collectors.toList());
+    // validators has to be initialized
+    if (validators == null) {
+      this.validators = null;
```

Comment:
Would it make more sense to initialize `this.validators` as an empty List? Unless the distinction between empty `validators` and null `validators` matters, I think that we could prevent complications down the line if we reduce the number of null values which our codebase permits.

Reply:
I do like the idea of empty list better. It seems the unit tests pass with empty list initialization so that seems to work. Initially I was trying to avoid the NPE.